scripts/convert-docs.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const { writeFile } = require('fs/promises');
+const apiDocs = require('../src/api-docs');
+
+(async () => {
+  // Remove `default` export property
+  const docsObj = apiDocs.default;
+
+  // Remove obsolete properties
```

Comment:
They are not obsolete, the explanation is in the other comment. We should discuss the best approach for that, maybe we should remove from the js docs, not here in the script.

Reply:
On [9f2ac73](https://github.com/HathorNetwork/hathor-wallet-headless/pull/327/commits/9f2ac7363b8f9c730d99feeac429be93197554c7) a check was implemented to fetch the API Key configuration directly from the config file at runtime.

No arbitrary removal is necessary anymore.